# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/OffscreenCanvas.cpp
# Commit: db2347ee8147
# Full Hash: db2347ee81479323d8b188df53a9cb3c0ad6c6e7
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2021-12-10 09:10:34
# Regressor Bug: 1736177
# File Overlap Count: 2
# Description:
#   Bug 1736177 - Part 1. Rename CanvasRenderingContextHelper::GetContext to GetOrCreateContext. r=jgilbert
#   
#   This patch contains no functional changes but sets us up for refactoring
#   in a later part in this series. It also helps silence a bunch of
#   compiler warnings.
# ==============================================================================

diff -r 3dde5ddb65e5 -r db2347ee8147 dom/canvas/OffscreenCanvas.cpp
--- a/dom/canvas/OffscreenCanvas.cpp	Thu Dec 09 19:25:23 2021 +0000
+++ b/dom/canvas/OffscreenCanvas.cpp	Thu Dec 09 19:25:24 2021 +0000
@@ -111,8 +111,8 @@
     return nullptr;
   }
 
-  RefPtr<nsISupports> result = CanvasRenderingContextHelper::GetContext(
-      aCx, aContextId, aContextOptions, aRv);
+  RefPtr<nsISupports> result = CanvasRenderingContextHelper::GetOrCreateContext(
+      aCx, contextType, aContextOptions, aRv);
 
   if (!mCurrentContext) {
     return nullptr;